### DISCUSSION TITLE
Umbraco v14 plan or timeline?

### DISCUSSION THREAD

**Discussion by nackler** [skartknet/ConditionalDisplayers#29] [Q&A]
Trying to plan my migration to Umbraco 14 and wondering if there are plans to build an Umbraco 14 version of this plugin. If not, then I will wean myself off of it. Fairly new to umbraco so not quite ready to tackle building a replacement for this on my own.

**Comment by skartknet** [comment]
Hi @nackler , yes I have been trying to find a way to do it on v14 but still can't find the way.

V14 is totally different and it doesn't include the data attributes that CD uses to find each property. 

I can't tell you when it's going to be ready. I just don't know.

**Reply by nackler** [reply]
No worries. Sounds like quite a paradigm shift. Good luck figuring it out.

**Comment by JohanReitsma83** [comment]
Hi @skartknet are you getting somewhere with this awesome package?

**Comment by skartknet** [comment]
Hey @JohanReitsma83 , unfortunately not yet. I'll keep you in the loop when I have any more news.

**Comment by skartknet** [comment]
I will close this discussion as we are already in version 16 😃
